### ISSUE TITLE
Is it Possible to change the start direction of products in slider? from right side

### ISSUE THREAD

**Issue by tayyabroy** [OwlCarousel2/OwlCarousel2#1893]
Is it Possible to change the start direction of products in slider? from right side
please check attache Image.

![screenshot- domain - date - time](https://cloud.githubusercontent.com/assets/28806886/26247568/e6f5fe56-3cb7-11e7-8fef-f493cc41f38b.png)

**Comment by keefyhub** [comment]
yep, use `startPosition` - http://owlcarousel2.github.io/OwlCarousel2/docs/api-options.html#startposition

You can either set the number by doing something like `$('.owl-item').length` or give it a hash ID

**Comment by tayyabroy** [comment]
$('.owl-carousel').owlCarousel({
    center: true,
    items:4,
    loop:false,
   startPosition: 'right'
    responsive:{
        600:{
            items:4
        }
    },
});


it's not working ?? I want to show product start from right as it in above image on left side..

**Comment by keefyhub** [comment]
http://codepen.io/keefyboooo/pen/mmLdXO

You need to specify a number or hash ID to start. so you could either add the #last-slide to the last slide in your HTML, or use some javascript like I have in the codepen above

**Comment by tayyabroy** [comment]
yes! thats great..
Thanks :)
